ad400x-fmcz/ad400x_fmcz.c

```diff
@@ -0,0 +1,117 @@
+#include <stdio.h>
+#include <sleep.h>
+#include <stdbool.h>
+#include <stdint.h>
+#include <stdlib.h>
+#include <xil_cache.h>
+#include <xparameters.h>
```

Comment:
i wonder if we should start a new folder called `eval-boards` or something like that and put this there, together with all existing eval boards

Reply:
let's leave it for later